index.bs

```diff
@@ -1199,6 +1217,11 @@ Note: Setting {{WebTransport/anticipatedConcurrentIncomingUnidirectionalStreams}
 {{WebTransport/anticipatedConcurrentIncomingBidirectionalStreams}} does not guarantee
 the application will receive the number of streams it anticipates.
 
+: <dfn for="WebTransport" attribute>headers</dfn>
+:: Once a [=WebTransport session=] has been established, holds the application-level
+   headers set by the server, if any. Initially null.
+   The getter steps are to return [=this=]'s {{[[Headers]]}}.
```

Comment:
This is a little hand-wavy. Shouldn't you define somewhere a slot that holds null or a Headers object?